client/my-sites/plans/plan-overview/style.scss

```diff
@@ -0,0 +1,27 @@
+@mixin plan-overview-button {
```

Comment:
This approach makes sense too, but I didn't like the idea of having rules for one component live inside another. I wonder what @mtias thinks...